src/docs/extend/acf-fields.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: ACF fields.
+description: This page will describe the recommended process of adding additional field groups using ACF builder.
+---
+
+Although it's possible to add or modify ACF fields using the build in UI method, we highly recommend adding the fields using PHP configurations. We find this method more robust, especially in larger websites.
+If you want to see all the different field types that exist and their default atrtributes you can take a look right [here](https://github.com/StoutLogic/acf-builder/wiki/field-types).
+
+To add or modify ACF fields using PHP configurations you need a fieldgroup. Within WP-lemon some post types will already have an existing fieldgroup. 
+In the following documentation we will explain both how you can manage your ACF fields that have an existing fieldgroup, and how to create your own fieldgroup.
+
+
+
+
+## Modifying existing fieldgroups
+Because these post types will have a fieldgroup by default you don't have to create it yourself. 
+By navigating to the `libary/hooks.php` file in your child theme, you can use filters to add, modify or remove fields in an existing fieldgroup.
+
+below we've listed all the existing fieldgroups with the filter name you can use.
+
+| **Fieldgroups**          | **Fieldgroups filter name**    |
+|------------------------|------------------------|
+| Job                    | wp-lemon/filter/model/acf-fields/job  |
+| Person                 | wp-lemon/filter/model/acf-fields/person  |
+| Menu                   | wp-lemon/filter/model/acf-fields/menu  |
+
+
+### Adding ACF fields
+```php 
+add_filter(
+   'wp-lemon/filter/model/acf-fields/job',
+   function ($job) {
+      $job
+         ->addText('job_location');
+      return $job;
+   },
+);
+```
+
+### Remove ACF Fields
+In the same way we can remove an existing ACF field by using the build in function that ACF provides like so:
+```php
+add_filter(
+   'wp-lemon/filter/model/acf-fields/job',
+   function ($job) {
+      $job
+         ->removeField('salary');
+      return $job;
+   },
+);
+```
+
+### Remove all ACF Fields at once
+In case you want to remove the whole specific fieldgroup of a post type to start from scratch you can return `false`.
```

Comment:
```suggestion
In case you want to remove the whole specific field group. you can return `false` in the filter.
You can use this option to completely build a field group from the ground up.
```